New package: MacroEnergy v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: MacroEnergy
- Repository: https://github.com/macroenergy/MacroEnergy.jl
- Created by: @RuaridhMacd
- Version: v0.0.1
- Commit: a47abcefb90707ed055e230b4ff66a7a51838e7b
- Reviewed by: @RuaridhMacd
- Reference: https://github.com/macroenergy/MacroEnergy.jl/commit/a47abcefb90707ed055e230b4ff66a7a51838e7b#commitcomment-164341002
- Description: Macro allows you to optimize the design and operations of large multi-sector energy systems.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6845db7639c7ce9a0a15332d6ecfd510d7ccdbaee41765130dc185a48638e50c03292826fa2b18bf9cebc1f924c5564d3d42dd75a733f5711701a11c346454b3dd8ddd793400813be299acbd08f8ed025b8b5dc2f508d41fd22147f310da885b2adc9c3ab96405e20a88f928ede314928ba0289d04f196e460bb2aaf734bbf92af88e081861190647983edf2dff8212fe7d29b57616fab600e5895a61d58654a0f029154905bfb72efeff91763e7d3e7e -->

goerz commented:
[noblock] It looks like you may have renamed this from `Macro` to `MacroEnergy`, but there are still a lot of references to `Macro` in the README and the documentation. Definitely make sure to fix those as soon as you can (before any `v0.1` or `v1.0` release)

RuaridhMacd commented:
> [noblock] It looks like you may have renamed this from `Macro` to `MacroEnergy`, but there are still a lot of references to `Macro` in the README and the documentation. Definitely make sure to fix those as soon as you can (before any `v0.1` or `v1.0` release)

[noblock] Thanks. The working title has been "Macro" for the past year, with a longer package name. We're working to clean it up and figure out what's least confusing for new users.

JuliaRegistrator committed:
New package: MacroEnergy v0.0.1

UUID: 260b7737-bf0e-4e84-9335-3af14438b1f5
Repo: https://github.com/macroenergy/MacroEnergy.jl.git
Tree: 583d66bb20effde33b23ab27b939d3a2a6dde700

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
